src/readableview.js

```diff
@@ -0,0 +1,67 @@
+/** @flow */
```

Comment:
The [Google JS Style Guide](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Naming__body) says:

> In general, use ... filenameslikethis.js.
> Filenames should be all lowercase in order to avoid confusion on case-sensitive platforms. Filenames should end in .js, and should contain no punctuation except for - or _ (prefer - to _).

So it should be either `readableview.js` or `readable-view.js`. Any preference?